br.app.pw3270.terminal.yml

```diff
@@ -0,0 +1,54 @@
+app-id: br.app.pw3270.terminal
+runtime: org.gnome.Platform
+runtime-version: 43
+sdk: org.gnome.Sdk
+command: pw3270
+finish-args:
+  - "--share=network"
+  - "--share=ipc"
+  - "--socket=fallback-x11"
+  - "--socket=wayland"
+  - "--own-name=br.app.pw3270.pw3270.terminal"
+cleanup:
+  - "/include"
+  - "/lib/pkgconfig"
+  - "/man"
+  - "/share/doc"
+  - "/share/gtk-doc"
+  - "/share/man"
+  - "/share/pkgconfig"
+  - "*.la"
+  - "*.a"
+modules:
+  - name: lib3270
+    buildsystem: autotools
```

Comment:
autotools mean no build commands. Just in you case some `config-opt`. That's the idea behind build systems.

(this apply below)